scripts/install-mender.sh

```diff
@@ -270,16 +283,9 @@ do_install_commercial() {
 
     echo "  Installing commercial components from $MENDER_COMMERCIAL_DOWNLOAD_URL"
 
-    # Translate Debian "channel" into Mender version
-    if [ "$CHANNEL" = "experimental" ]; then
-        version="master"
-    else
-        version="latest"
-    fi
-
     # Download deb packages
     for c in $selected_components_commercial; do
-        url="$MENDER_COMMERCIAL_DOWNLOAD_URL$(printf ${COMMERCIAL_COMP_TO_URL_PATH_F[$c]} $version $version $LSB_DIST $DIST_VERSION)"
+        url="$MENDER_COMMERCIAL_DOWNLOAD_URL$(printf ${COMMERCIAL_COMP_TO_URL_PATH_F[$c]} $VERSION $VERSION $LSB_DIST $DIST_VERSION)"
```

Comment:
* c should be local
* url should be local
* $MENDER_COMMERCIAL_DOWNLOAD_URL -> `${MENDER_COMMERCIAL_DOWNLOAD_URL}`
*